packages/salesforcedx-vscode-core/test/jest/conflict/metadataCacheService.test.ts

```diff
@@ -0,0 +1,52 @@
+import { ComponentSet } from '@salesforce/source-deploy-retrieve';
+import { MetadataCacheService } from '../../../src/conflict';
+import { WorkspaceContext } from '../../../src/context';
+import * as sdrUtils from '../../../src/services/sdr/componentSetUtils';
+
+describe('MetadataCacheService', () => {
```

Comment:
Alright similar question here as for for timestampConflictDetector.test.ts under vscode-integration. That one would be a lot more to move over, so might not be the right time, but is it worth making any changes over there?